internal/pkg/aws/ecs/ecs.go

```diff
@@ -40,8 +84,96 @@ func (s Service) TaskDefinition(taskDefName string) (*TaskDefinition, error) {
 	return &td, nil
 }
 
-// TaskDefinition wraps up ECS TaskDefinition struct.
-type TaskDefinition ecs.TaskDefinition
+// Service calls ECS API and returns the specified service running in the cluster.
+func (e *ECS) Service(clusterName, serviceName string) (*Service, error) {
+	resp, err := e.client.DescribeServices(&ecs.DescribeServicesInput{
+		Cluster:  aws.String(clusterName),
+		Services: aws.StringSlice([]string{serviceName}),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("describe service %s: %w", serviceName, err)
+	}
+	for _, service := range resp.Services {
+		if aws.StringValue(service.ServiceName) == serviceName {
+			svc := Service(*service)
+			return &svc, nil
+		}
+	}
+	return nil, fmt.Errorf("cannot find service %s", serviceName)
+}
+
+// Tasks calls ECS API and returns ECS tasks running in the cluster.
+func (e *ECS) Tasks(clusterName, serviceName string) ([]*Task, error) {
```

Comment:
Maybe call this ServiceTasks since we might need to get non-service tasks in the future.

Reply:
Good idea #thinkbig#